### PR TITLE
Adoption of ABINet for ms 2.1

### PR DESCRIPTION
**Pull request by Gaohan123**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs
https://github.com/mindspore-lab/mindocr/issues/491

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

**Comment by HaoyangLee** [comment]
add ABINet yaml config file path here
https://github.com/mindspore-lab/mindocr/blob/main/tests/ut/test_models.py#L16

File: configs/rec/abinet/README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ According to our experiments, the evaluation results on public benchmark dataset
 
 | **Model** | **Context** | **Avg Accuracy** | **Train T.** | **FPS** | **Recipe** | **Download** |
 | :-----: | :-----------: | :--------------: | :----------: | :--------: | :--------: |:----------: |
-| ABINet      | D910x8-MS1.9-G | 92.53%    | 22993 s/epoch       | 628.11 | [yaml](abinet_resnet45_en.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/abinet/abinet_resnet45_en-41e4bbd0.ckpt)
+| ABINet      | D910x8-MS1.9-G | 91.35%    | 22993 s/epoch       | 628.11 | [yaml](abinet_resnet45_en.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/abinet/abinet_resnet45_en-7efa1184.ckpt)
```
Comment (liangxhao) [inline configs/rec/abinet/README.md:40]: 部分值和README_CN.md不一致

Comment (HaoyangLee) [reply configs/rec/abinet/README.md:40]: https://github.com/mindspore-lab/mindocr/blob/main/docs/cn/mkdocs/modelzoo_training.md
https://github.com/mindspore-lab/mindocr/blob/main/docs/en/mkdocs/modelzoo_training.md
精度性能数据同步更新到这两处文档

File: mindocr/data/transforms/rec_abinet_transforms.py [hunk 2]
```diff
@@ -102,12 +103,14 @@ def __init__(self):
         self.toTensor = ds.vision.ToTensor()
         self.w = 128
         self.h = 32
+        self.op = ms.dataset.vision.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225], is_hwc=False)
```
Comment (tonytonglt) [inline mindocr/data/transforms/rec_abinet_transforms.py:106]: 建议把resize的shape写成config中的可配置项。另外Normalize是否可以使用new pipeline的Normalize接口？

File: mindocr/data/transforms/rec_abinet_transforms.py [hunk 3]
```diff
@@ -171,11 +174,13 @@ def __init__(self):
...
 
     def __call__(self, data):
         img = data["image"]
         img = cv2.resize(img, (self.w, self.h))
         img = self.toTensor(img)
+        # img = self.op(img)
```
Comment (tonytonglt) [inline mindocr/data/transforms/rec_abinet_transforms.py:183]: 尽量不要保留注释掉的代码

File: mindocr/postprocess/rec_abinet_postprocess.py [hunk 4]
```diff
@@ -49,11 +49,9 @@ def __call__(self, preds: Union[Tensor, np.ndarray], labels=None, **kwargs):
             texts (List[Tuple]): list of string
 
         """
-        output = preds
-        output = self._get_output(output)
-        logits, pt_lengths = output[0], output[1]
+        logits = preds
         pt_text, pt_scores, pt_lengths_ = self.decode(logits)
-        assert (pt_lengths == pt_lengths_).all(), f"{pt_lengths} != {pt_lengths_} for {pt_text}"
+        # assert (pt_lengths == pt_lengths_).all(), f"{pt_lengths} != {pt_lengths_} for {pt_text}"
```
Comment (tonytonglt) [inline mindocr/postprocess/rec_abinet_postprocess.py:54]: 注释

**Comment by Gaohan123** [comment]
> add ABINet yaml config file path here https://github.com/mindspore-lab/mindocr/blob/main/tests/ut/test_models.py#L16

已修改

File: configs/rec/abinet/README.md (hunk 1 above)
Comment (Gaohan123) [reply configs/rec/abinet/README.md]: 已修改

File: mindocr/data/transforms/rec_abinet_transforms.py (hunk 2 above)
Comment (Gaohan123) [reply mindocr/data/transforms/rec_abinet_transforms.py:111]: resize已修改。新流程normalize涉及新data pipeline的适配，这里主要是适配2.1。后续可以再修改

File: mindocr/data/transforms/rec_abinet_transforms.py (hunk 3 above)
Comment (Gaohan123) [reply mindocr/data/transforms/rec_abinet_transforms.py]: 已修改

File: mindocr/postprocess/rec_abinet_postprocess.py (hunk 4 above)
Comment (Gaohan123) [reply mindocr/postprocess/rec_abinet_postprocess.py]: 已修改

**Review by tonytonglt — APPROVED** [review]

**Review by liangxhao — APPROVED** [review]

**Pull request merged**